src/test_lib_json/fuzz.cpp

```diff
@@ -44,7 +43,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
   std::unique_ptr<Json::CharReader> reader(builder.newCharReader());
 
   Json::Value root;
-  const char* data_str = reinterpret_cast<const char*>(data);
+  const auto data_str = reinterpret_cast<const char*>(data);
```

Comment:
`data_str` changed type here in a subtle way, from `const char*` to `const char* const`.